web/client/src/library/pages/docs/Search.tsx

```diff
@@ -91,57 +93,28 @@ function highlightMatch(source: string, match: string): JSX.Element {
 }
 
 function filterModelsBySearch(
-  indices: Array<[Model, string]> = [],
+  indices: Array<[ModelSQLMeshModel, string]> = [],
   search: string,
-): Array<[Model, string]> {
-  return indices.reduce((acc: Array<[Model, string]>, [model, index]) => {
-    const idx = index.indexOf(search.toLocaleLowerCase())
-
-    if (idx > -1) {
-      const SIZE = 40
-      const min = Math.max(0, idx - SIZE)
-      const max = Math.min(index.length - 1, idx + search.length + SIZE)
-
-      acc.push([
-        model,
-        (min > 0 ? '... ' : '') +
-          index.slice(min, max) +
-          (max < index.length ? ' ...' : ''),
-      ])
-    }
-
-    return acc
-  }, [])
-}
-
-function createIndices(models: Map<string, Model>): Array<[Model, string]> {
-  const indices: Array<[Model, string]> = []
-
-  models.forEach((value, key) => {
-    if (value.path === key) return
-
-    const index = Object.entries(value).reduce((acc, [k, v]) => {
-      if (k === 'sql') return acc
-
-      if (k === 'details' && v != null) {
-        return acc + ' ' + Object.values(v).join(' ')
-      }
-
-      if (k === 'columns' && v != null) {
-        return (
-          acc +
-          ' ' +
-          (v as Array<Record<string, unknown>>)
-            .map(column => String(Object.values(column).join(' ')))
-            .join(' ')
-        )
+): Array<[ModelSQLMeshModel, string]> {
+  return indices.reduce(
+    (acc: Array<[ModelSQLMeshModel, string]>, [model, index]) => {
+      const idx = index.indexOf(search.toLocaleLowerCase())
+
+      if (idx > -1) {
+        const SIZE = 40
```

Comment:
Should we move this out of the reduce? Maybe use a more descriptive name and make it a global variable?